conduit/data/datamodules/base.py

```diff
@@ -65,7 +65,7 @@ class CdtDataModule(pl.LightningDataModule, Generic[D, I]):
     """
 
     train_batch_size: int = 64
-    _eval_batch_size: Optional[int] = None
+    _eval_batch_size: Optional[int] = attr.field(alias="eval_batch_size", default=None)
```

Comment:
This change makes pyright understand that the argument name in the `__init__` is `eval_batch_size`.